ush/python/pygfs/task/marine_letkf.py

```diff
@@ -133,6 +148,32 @@ def run(self):
 
         logger.info("run")
 
+        exec_cmd_gridgen = Executable(self.task_config.APRUN_MARINEANALLETKF)
+        exec_cmd_gridgen.add_default_arg(self.task_config.GRIDGEN_EXEC)
+        exec_cmd_gridgen.add_default_arg(self.task_config.GRIDGEN_YAML)
+
+        try:
```

Comment:
I know this will be refactored soon, but maybe use this in the meantime [marine_da_utils.py#L18](https://github.com/NOAA-EMC/global-workflow/blob/c6e32621a71e71d250c64c5349d9249cfeec2cf4/ush/python/pygfs/utils/marine_da_utils.py#L18) ?

Reply:
Why not

Reply:
Add an issue and maybe also a TODO if you don't want to address this in this PR. This will be refactored when we start making use of the Jedi class.

Reply:
changed - I forgot about this comment